dataactcore/models/jobTrackerInterface.py

```diff
@@ -6,18 +6,12 @@
 from dataactcore.utils.statusCode import StatusCode
 from dataactcore.utils.responseException import ResponseException
 from dataactcore.utils.cloudLogger import CloudLogger
-from dataactcore.utils.jobQueue import JobQueue
-from dataactcore.config import CONFIG_JOB_QUEUE
+from dataactcore.utils.jobQueue import enqueue
 from dataactvalidator.validation_handlers.validationError import ValidationError
 
 
 class JobTrackerInterface(BaseInterface):
     """Manages all interaction with the job tracker database."""
-
-    def __init__(self):
-        self.jobQueue = JobQueue(job_queue_url=CONFIG_JOB_QUEUE['url'])
-        super(JobTrackerInterface, self).__init__()
```

Comment:
Do we lose anything crucial by no longer invoking the `__init__` of the parent class here (_i.e._, `BaseInterface`). I'm asking because I see that `BaseInterface` sets `self.dbName`, which is maaaybe used in the `baseTestAPI.py` and `baseTestValidator.py` teardown functions. 

That said, everything seems to be working here, test-wise, so perhaps I got lost in the interfaces.